resource-managers/kubernetes/integration-tests/src/test/scala/org/apache/spark/deploy/k8s/integrationtest/Utils.scala

```diff
@@ -49,4 +60,39 @@ object Utils extends Logging {
     out.flush()
     out.toString()
   }
+
+  def createTempFile(contents: String, hostPath: String): String = {
+    val filename = try {
+      val f = File.createTempFile("tmp", ".txt", new File(hostPath))
+      f.deleteOnExit()
+      new PrintWriter(f) {
+        try {
+          write(contents)
+        } finally {
+          close()
+        }
+      }
+      f.getName
+    } catch {
+      case e: Exception => e.printStackTrace(); throw e;
+    }
+    filename
+  }
+
+  def getExamplesJarAbsolutePath(sparkHomeDir: Path): String = {
+    val jarName = getExamplesJarName()
+    val jarPath = Files.walk(sparkHomeDir).filter(Files.isRegularFile(_)).filter((f: Path) => {
+      f.toFile.getName == jarName
+    })
+      .iterator()
```

Comment:
The continuation looks a little weird; introduce a val?

Reply:
sure